src/box/index.cc

```diff
@@ -631,29 +629,18 @@ iterator_position(struct iterator *it, const char **pos, uint32_t *size)
 	return it->position(it, pos, size);
 }
 
-/** Keys for iterator position map. All the keys must be uint. */
-enum {
-	ITERATOR_POSITION_KEY = 0,
-};
+static const int iterator_pos_b64_opts = BASE64_NOPAD | BASE64_NOWRAP;
 
 uint32_t
-iterator_position_pack_size(const char *pos, const char *pos_end,
-			    uint32_t part_count)
+iterator_position_pack_size(const char *pos, const char *pos_end)
```

Comment:
AFAIU It's not an *exact* pack size - it's the *maximal* size of packed data. If so, it's better to name this function `iterator_position_pack_size_max`.

Reply:
BTW if pack/unpack have a buf argument, it'd probably be better to name these functions `iterator_position_pack_bufsize`, `iterator_position_unpack_bufsize` to emphasize what they are supposed to be used for.

Reply:
Done